web/pdf_find_controller.js

```diff
@@ -120,16 +164,41 @@ var PDFFindController = (function PDFFindControllerClosure() {
         query = query.toLowerCase();
       }
 
-      var matches = [];
-      var matchIdx = -queryLen;
-      while (true) {
-        matchIdx = pageContent.indexOf(query, matchIdx + queryLen);
-        if (matchIdx === -1) {
-          break;
+      if (phraseSearch) {
+        subqueryArray = [query];
```

Comment:
For "normal" searching this will always allocate an array with `length === 1`, which seems unnecessary. How about [moving this code](https://github.com/jazzy-em/pdf.js/blob/master/web/pdf_find_controller.js#L173-L190) into an inline function instead? E.g.
```javascript
function addMatches(query) {
  // The code linked above is placed here...
}
```
Then you could do:
```javascript
if (phraseSearch) {
  addMatches(query);
} else {
  var queryArray = query.split(DIVIDER_TERMS);
  queryArray.forEach(addMatches);
}
```